Resources/Views/Submissions/Fields/textarea-input.leaf

```diff
@@ -3,7 +3,7 @@
         <label for="#(key)">#(label)</label>
     }
 
-    <textarea class="form-control #if(hasErrors) { is-invalid }" id='#(key)' name='#(key)' cols="" rows="6" #if(placeholder) { placeholder='#(placeholder)' } #if(isRequired) { required }>#if(value) { #(value) }</textarea>
+    #bs:textArea("form-control #if(hasErrors) { is-invalid }", "id='#(key)' name='#(key)' cols='' rows='6' #if(isRequired) { required } #if(placeholder) { placeholder='#(placeholder)' }", "#(value)")
```

Comment:
I see you replaced some `"`s with `'` but not everywhere, is there a reason?

Reply:
The old wasn't consistent but the new one should be. "" for the leaf parameters - '' for the html values being set (inside the parameter given to Leaf).